helm/cluster-api-provider-azure/templates/webhook.yaml

```diff
@@ -11,9 +11,10 @@ webhooks:
 - clientConfig:
     caBundle: Cg==
     service:
-      name: {{ include "resource.default.name" . }}-webhook
+      name: {{ include "resource.default.name" . }}
       namespace: {{ include "resource.default.namespace" . }}
       path: /mutate-infrastructure-cluster-x-k8s-io-v1alpha3-azurecluster
```

Comment:
Note for myself: These paths need checking. I _think_ they should be matching the current GVK?